.github/workflows/python-package.yaml

```diff
@@ -37,6 +37,14 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-poetry-
 
+      # TODO(ckuehl|2024-09-26): Remove this once Python 3.13 wheels are
+      # available for our dependencies.
+      - name: Install temporary Python 3.13 build dependencies
+        if: ${{ matrix.python-version == '3.13' }}
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
Call me old fashioned, but it seems like a good idea to install a specific version and verify checksums.

Reply:
👍  yeah how long do we expect this workaround to be in place?